README.md

```diff
@@ -66,6 +66,18 @@ A local dev server will be created on
 
 Flask environment variables are configurable in `.flaskenv`
 
+# Run with Gunicorn
+
+In deployed environments the service is run with gunicorn. You can run the service locally with gunicorn to test
+
+First set the FLASK_ENV environment you wish to test eg:
+
+    export FLASK_ENV=dev
```

Comment:
im wondering if this should be stated outside of the 'Run with Gunicorn' section as it applies to running in Flask also